app/portainer/views/support/product/productController.js

```diff
@@ -3,11 +3,36 @@ angular.module('portainer.app')
     function($scope, $transition$) {
 
       $scope.formValues = {
-        hostCount: 10
+        hostCount: 0
+      };
+
+      $scope.state = {
+        minHosts: 0,
+        placeholder: 0
+      };
+
+      $scope.buttonChecker = function () {
```

Comment:
Even if it's all technically looking OK, I just have a comment on naming. Always better/clearer to have a proper function name to understand what this function is about without having to use comments.

I'd rename this to `isBuyButtonEnabled`.

Reply:
Great point! Made the change